### ISSUE TITLE
Can you offer your paper?

### ISSUE THREAD

**Issue by Grandzxw** [bobocode/lmono#1]
Thank you for your great work! can you offer your paper?and when I run the KITTI 00, it meet the follow error:
`receiving laser odometry data 0.103736
mearure img with stamp 0.103736
measure laser with stamp 0.103736
[ INFO] [1635953725.761876823]: measurements obtained: 1
[ INFO] [1635953725.761972402]: processing laser data with stamp 103735901
[monolio_estimator-1] process has died [pid 27184, exit code -11, cmd /home/grand/V-LOAM/devel/lib/monolio/monolio_estimator_node __name:=monolio_estimator __log:=/home/grand/.ros/log/815a8fe0-3cbb-11ec-9edc-709cd1e76251/monolio_estimator-1.log].
log file: /home/grand/.ros/log/815a8fe0-3cbb-11ec-9edc-709cd1e76251/monolio_estimator-1*.log
all processes on machine have died, roslaunch will exit`

How to run this code?

**Comment by bobocode** [comment]
Many thanks for your appreciation. can you post more logs when you run the code? it seems that your process exit suddenly. With limited information, i cannot sure what problem you have encountered.

**Comment by Grandzxw** [comment]
> Many thanks for your appreciation. can you post more logs when you run the code? it seems that your process exit suddenly. With limited information, i cannot sure what problem you have encountered.

YES, First I open a terminal to run "**roslaunch aloam_velodyne aloam_velodyne_HDL_64.launch**", then I open another a terminal to run " **roslaunch monolio kitti_estimator_00.launch**". The terminal of aloam is OK, but the monolio's terminal was met error, and the log is as follow:
`... logging to /home/grand/.ros/log/4da94458-3cbf-11ec-9edc-709cd1e76251/roslaunch-grand-2432.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://grand:40789/

SUMMARY
========

PARAMETERS
 * /monolio_estimator/config_file: /home/grand/V-LOA...
 * /rosdistro: melodic
 * /rosversion: 1.14.11

NODES
  /
    monolio_estimator (monolio/monolio_estimator_node)

ROS_MASTER_URI=http://localhost:11311

process[monolio_estimator-1]: started with pid [2457]
[ INFO] [1635955301.039570113]: initial beigins
[ INFO] [1635955301.040301290]: Loaded config_file: /home/grand/V-LOAM/src/lmono-master/mono_lidar_mapping/config/kitti_config_01.yaml
image0 topic: /image_left
image1 topic: /image_right
cam0 calib: /home/grand/V-LOAM/src/lmono-master/mono_lidar_mapping/config/kitti00_cam.yaml
cam1 calib: /home/bo/lmono/src/mono_lidar_mapping/config/kitti_cam01.yaml
laser to cam0: 
0.000427683 -0.00721068    0.999974    0.288779
  -0.999967  0.00808118 0.000485952  -0.0554166
-0.00808447   -0.999941 -0.00720698   0.0454265
          0           0           0           1
cam0 to cam1: 
       1        0        0 0.537151
       0        1        0        0
       0        0        1        0
       0        0        0        1
cam[0] calib information: 
718.856       0 607.193       0
      0 718.856 185.216       0
      0       0       1       0
camera file does not exist.cam[1] calib information: 
           0            0            0 4.68682e-310
           0            0 9.88131e-324 8.89318e-323
           0            0            1            0
receiving laser odometry data 6.842350
receiving laser odometry data 6.945963
mearure img with stamp 6.945963
measure laser with stamp 6.945963
[ INFO] [1635955301.422596268]: measurements obtained: 1
[ INFO] [1635955301.422709768]: processing laser data with stamp 6945962906
receive the lidar pose is:    0.997009  -0.0769612  0.00707146     60.8314
  0.0769387     0.99703  0.00340489     3.74716
 -0.0073125 -0.00285063    0.999969    0.171898
          0           0           0           1
[ INFO] [1635955301.435103386]: processing image dara with stamp 6945962906
[ INFO] [1635955301.435148324]: >>>>>>>>>>>>>>>>>>>new image coming<<<<<<<<<<<<<<<<<<
[ INFO] [1635955301.454151773]: feature tracker time: 18.4747
long track num: 0
last track num: 0
new feature num: 150
keyframe
[monolio_estimator-1] process has died [pid 2457, exit code -11, cmd /home/grand/V-LOAM/devel/lib/monolio/monolio_estimator_node __name:=monolio_estimator __log:=/home/grand/.ros/log/4da94458-3cbf-11ec-9edc-709cd1e76251/monolio_estimator-1.log].
log file: /home/grand/.ros/log/4da94458-3cbf-11ec-9edc-709cd1e76251/monolio_estimator-1*.log
all processes on machine have died, roslaunch will exit
shutting down processing monitor...
... shutting down processing monitor complete
done
`
